<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRCoder</class>
 <widget class="QMainWindow" name="QRCoder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <widget class="QLineEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>271</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="textLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Data</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="scaleSpinBox">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>0</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
      <widget class="QComboBox" name="encModeComboBox">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>0</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="scanDistanceLabel">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>300</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string> D (cm)</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="scanDistanceSpinBox">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>300</y>
         <width>41</width>
         <height>22</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Distance in Centimeters QR code is expected to scan (1in=2.54cm)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>31</number>
       </property>
      </widget>
      <widget class="QLabel" name="screenDPILabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>320</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Screen DPI:</string>
       </property>
      </widget>
      <widget class="QLabel" name="dpiLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>320</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>DPI:</string>
       </property>
      </widget>
      <widget class="QLabel" name="sizeLabel">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>320</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Size:</string>
       </property>
      </widget>
      <widget class="QLabel" name="distanceLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>300</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Distance:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="ecModeComboBox">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>0</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>663</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionPrint"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as ...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
